topi/python/topi/cpp/impl.py

```diff
@@ -32,7 +32,7 @@ def _get_lib_names():
 def _load_lib():
     """Load libary by searching possible path."""
     curr_path = os.path.dirname(os.path.realpath(os.path.expanduser(__file__)))
-    lib_search = curr_path
+    lib_search = [curr_path, os.path.join(curr_path, "..")]
```

Comment:
you can use `os.path.dirname(` to get the parent folder
```
>>> import os
>>> os.path.dirname('/tmp')
'/'
```

Reply:
Sure. Thanks for the pointer :-)